### PR TITLE
Removed old code.

### PR DESCRIPTION
**Pull request by diegoreymendez**
Removes old unused code until the new services are ready to replace the old ones.

**How to test:**
- Make sure the code builds properly.

/cc @kwonye

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Themes/ThemeBrowserViewController.m [hunk 1]
```diff
@@ -135,6 +139,7 @@ - (void)syncThemesAndCurrentTheme
             [_refreshHeaderView endRefreshing];
         } failure:failureBlock];
     } failure:failureBlock];
+     */
```
Comment (kwonye) [inline WordPress/Classes/ViewRelated/Themes/ThemeBrowserViewController.m]: I would prefer we just delete this code vs commenting it out.

File: WordPress/Classes/ViewRelated/Themes/ThemeDetailsViewController.m [hunk 2]
```diff
@@ -256,6 +260,7 @@ - (IBAction)activatePressed:(id)sender
         [self.activateButton setTitle:NSLocalizedString(@"Activate", nil) forState:UIControlStateNormal];
         [WPError showNetworkingAlertWithError:error];
     }];
+     */
```
Comment (kwonye) [inline WordPress/Classes/ViewRelated/Themes/ThemeDetailsViewController.m]: Same comment about deletion

**Comment by diegoreymendez** [comment]
Ready for another look.

@jleandroperez would you mind taking over as @kwonye is on vacation?

**Comment by jleandroperez** [comment]
YAY for code deletion!

:shipit:

**Pull request merged**